packages/polly-request-presigner/src/getSynthesizeSpeechUrl.ts

```diff
@@ -0,0 +1,35 @@
+import { Polly, PollyClient, SynthesizeSpeechCommand, SynthesizeSpeechInput } from "@aws-sdk/client-polly";
+import { getSignedUrl } from "./getSignedUrls";
+
+export interface PresignedPollyOptions {
+  /**
+   * Seconds until the presigned URL expires. Defaults to 900
+   */
+  expiresIn?: number;
+}
+
+/**
+ * Generate a signed URL.
+ */
+export interface PresignedPollyInput {
+  /**
+   * A pre-configured instance of the AWS.Polly service object to use for requests. The object may bound parameters used by the presigner.
+   */
+  client: Polly | PollyClient;
```

Comment:
I realize `Polly` extends `PollyClient`. So maybe just `Polly`?

Reply:
You mean just `PollyClient`?

Reply:
Update to `pollyclient`